packages/java/parser-jvm-plugin-nonnull/src/main/java/dev/hilla/parser/plugins/nonnull/NonnullPlugin.java

```diff
@@ -51,29 +52,35 @@ public void enter(NodePath<?> nodePath) {
 
     @Override
     public void exit(NodePath<?> nodePath) {
-        if (!(nodePath.getNode().getTarget() instanceof Schema)) {
-            return;
-        }
-
-        var schema = (Schema<?>) nodePath.getNode().getTarget();
-
-        // Apply annotations from package (NonNullApi)
-        var annotations = getPackageAnnotationsStream(nodePath);
-
-        // Apply from current node, if source is annotated
-        if (nodePath.getNode().getSource() instanceof AnnotatedModel) {
-            annotations = Stream.concat(annotations,
-                    ((AnnotatedModel) nodePath.getNode().getSource())
-                            .getAnnotationsStream());
+        var node = nodePath.getNode();
+        if (node.getTarget()instanceof Schema schema) {
```

Comment:
There is a formatting rule concerning `instanceof` that is not working correctly or is just wrong.